playbooks/deploy_web_metacpan.yml

```diff
@@ -0,0 +1,24 @@
+# vim: set filetype=ansible.yaml:
```

Comment:
Have you got something in your `vim` config that treats Ansible config files differently?

Reply:
Yeah, there’s an ansible-lint plugin for vim/ale that I use that should pick up on that.